content/pages/disability-benefits/claims-appeal/hearings.md

```diff
@@ -6,63 +6,126 @@ plainlanguage: 11-4-16 certified in compliance with the Plain Writing Act
-<h5>Rescheduling Travel Board or Videoconference Hearings</h5>
-<p>If you asked for a Travel Board or videoconference hearing with the Board, you must file a motion if you want to request that the hearing be rescheduled. This needs to happen at least 2 weeks before the scheduled hearing date. You can do this by sending a letter that explains you have “good cause” to reschedule. “Good cause” includes situations in which you, your representative, or a witness are sick, or you have difficulty getting records needed for your appeal. A VLJ will review your motion and decide if the hearing can be rescheduled. You’ll get a copy in the mail of the decision about rescheduling your hearing.</p>
-<p>
-<em>Any request to reschedule must be in writing and be mailed to:</em><br/>
+**Learn more about:**
+
+[Scheduling your Board hearing](#scheduling)<br>
+[Preparing for your Board hearing](#preparing)<br>
+[What to expect at your Board hearing](#expect)
+
+<span id="scheduling"></span>
+
+## Scheduling your Board hearing
+
+### How do I request a Board hearing?
+
+You can request a Board hearing when you fill out your VA Form 9.<br>
+
+**The form will ask you to choose 1 of these 4 options:**
+  -	**No hearing.** If you choose this option, you can send a letter to the Board saying what you would have said at a hearing.
+  -	**A videoconference hearing at your local VA office.** This takes place at your local VA office (or another facility close to you). VA can usually schedule video hearings sooner than in-person hearings.<br>
+  [Find out more about choosing a video hearing](https://www.bva.va.gov/docs/BVA-VideoHearing-508version.pdf).
+  -	**An in-person hearing at your local VA office.** This can sometimes delay your hearing, because judges have to be scheduled to travel to your local VA office.
+  -	**An in-person hearing at the Board in Washington, DC.**
+
+[Download VA Form 9](https://www.va.gov/vaforms/va/pdf/va9.pdf).
+
+Or, you can contact your Veterans Service Organization to get help with requesting a hearing at any time during the appeals process.<br>
+[Find a Veterans Service Organization](https://www.va.gov/vso/).
+
+### How will I know when my hearing is scheduled?
+
+You’ll receive a notice in the mail at least 30 days before your hearing is scheduled.
+
+### What if I need to reschedule my hearing?
+
+Send a written request to reschedule your hearing at least 2 weeks before your scheduled hearing. Include your name and the VA file number for your claim.
+
+**Mail your request to:**
```

Comment:
I think the convention is to make this a heading (h4 or h5). @bethpotts to confirm.